common/src/main/java/com/microsoft/identity/common/internal/commands/DeviceCodeFlowCommand.java

```diff
@@ -38,6 +40,16 @@
  * exception handling.
  */
 public class DeviceCodeFlowCommand extends TokenCommand {
+
+    public final static String AUTH_DECLINED_CODE = "authorization_declined";
+    public final static String AUTH_DECLINED_MSG = "The end user has denied the authorization request. Re-run the Device Code Flow Protocol with another user.";
```

Comment:
Double check these error strings with @hamiltonha

Reply:
The error codes or the error messages?

Reply:
error messages.

Reply:
Ok, I'll be sure to sync up with @hamiltonha as I add more error messages in the next PR.